start_control_ssm.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+log() {
+  echo "$*" >&2
+}
+
+enable_hybrid_env_ssm() {
+  # SSM parameters necessary to register with a hybrid activation
+  activation_code=$(fetch_from_json '."ssm"."activation-code"' "${USER_DATA}")
+  activation_id=$(fetch_from_json '."ssm"."activation-id"' "${USER_DATA}")
+  region=$(fetch_from_json '."ssm"."region"' "${USER_DATA}")
```

Comment:
```suggestion
  activation_code=$(fetch_from_json '.ssm."activation-code"' "${USER_DATA}")
  activation_id=$(fetch_from_json '.ssm."activation-id"' "${USER_DATA}")
  region=$(fetch_from_json '.ssm.region' "${USER_DATA}")
```
nit: turns out it's the hyphen that needs quoting.  I'd vote for using the simpler unquoted form where possible, and just quoting the hyphenated parts, to make it clearer that that's why the extra quotes are there.

Another option is square bracketing everything to make it consistent, with the quotes standing out a little less (e.g. `.["ssm"]["activation-id"]`) but since plain dots are most common I'd vote to use them where possible, and quote where not, but it's not a big deal.

Reply:
I would prefer `.["ssm"]["activation-id"]` to `'.ssm."activation-id"'` or `'."ssm"."activation-id"'` since it is consistent with the admin container and also seems less fragile to tweaks (ie. someone changes the query slightly x years from now), if you are ok with it.